### PR TITLE
Manual maintenance sonar issues 

### PR DESCRIPTION
**Pull request by juliacialkowski**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [ ] A peer-review has been executed
  - [ ] The code has been manually inspected by someone who did not implement the feature
  - [ ] If this PR includes work on the frontend, at least one `@ts-nocheck` is removed. Additionally, there should not be any `@ts-nocheck` in files modified by this PR. If no `@ts-nocheck` are left: Celebrate :tada: :confetti_ball: type-safety and remove this entry. 
- [ ] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [ ] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [ ] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [ ] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [ ] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team

### CODE REVIEW

File: dataland-frontend/src/components/PredefinedComponentOptionsGetter.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { humanizeStringOrNumber } from '@/utils/StringFormatter';
  * @param componentName the name of the vue component to generate standard options for
  * @returns a set of pre-defined options for the defineCopmonent method
  */
-export function getComponentOptionsForDatalandInfoCard(componentName: string): ComponentOptionsWithObjectProps {
+export function getComponentOptionsForDatalandInfoCard(componentName: string): ComponentOptions {
```
Comment (SiegfriedSobkowiak) [inline dataland-frontend/src/components/PredefinedComponentOptionsGetter.ts:12]: Die Funktion scheint mir nicht genutzt zu werden.

Comment (juliacialkowski) [reply dataland-frontend/src/components/PredefinedComponentOptionsGetter.ts:12]: Sehe ich jetzt auch. Soll die Component entfernt werden ?

Comment (SiegfriedSobkowiak) [reply dataland-frontend/src/components/PredefinedComponentOptionsGetter.ts:12]: ja

Comment (juliacialkowski) [reply dataland-frontend/src/components/PredefinedComponentOptionsGetter.ts]: Habe ich.

**Pull request merged**